traits/trait_types.py

```diff
@@ -3652,7 +3650,7 @@ def resolve_class(self, object, name, value):
 
 #: List of container type values; default value is ``[]``. This trait type is
 #: deprecated. Use ``List(This(allow_none=False))`` instead.
-ListThis = List(ThisClass)
+ListThis = List(This(allow_none=False))
```

Comment:
Was this change intentional / necessary? Is this the result of `ThisClass` not being easily accessible?

Reply:
Yes, it was because `ThisClass` was the only remaining direct import from `trait_handlers` and the comment suggested that the substitution would be equivalent.